user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -488,11 +488,11 @@ cmake --build . --target install --config Debug
 cmake --build . --target install --config Release
 ----
 
-Although both the boost:build[] and the CMake scripts work fine, the final installed libraries will have different layouts for historical reasons.
+Although both the boost:build[B2] and the CMake scripts work fine, the final installed libraries will have different layouts for historical reasons.
 Both the CMake integration scripts and the libraries will be installed under different paths and this can create incompatibilities between Boost installations.
 
 TIP: You don't need to *build* Boost with CMake to be able to *use* Boost with CMake.
-CMake integration will work fine even if you install Boost with boost:build[].
+CMake integration will work fine even if you install Boost with boost:build[B2].
```

Comment:
Rene also requested the links to point somewhere else.

Reply:
@alandefreitas @grafikrobot 
From the Issue looks like boost:build[] should be replaced with:    https://www.bfgroup.xyz/b2/[B2]
I have done this for the getting started section - and seems to work - but please verifiy ....

Reply:
Yes. That looks correct.